mypy/nodes.py

```diff
@@ -117,13 +116,21 @@ def accept(self, visitor: NodeVisitor[T]) -> T:
         raise RuntimeError('Not implemented')
 
 
-# These are placeholders for a future refactoring; see #1783.
-# For now they serve as (unchecked) documentation of what various
+# This is a placeholder for a future refactoring; see #1783.
+# For now it serves as (unchecked) documentation of what various
 # fields of Node subtypes are expected to contain.
-Statement = Node
 Expression = Node
 
 
+class Statement(Node):
+    pass
+
+
+# For now, only Var is a declaration
```

Comment:
Curious what other uses you are planning? I am hesitant to add scaffolding for things we *might* need in the future. In practice the future always evolves in an unanticipated direction.